.github/workflows/deploy-poller.yml

```diff
@@ -54,26 +54,42 @@ jobs:
         if: env.SKIP_RUN == 'false'
         run: |
           # Define the URL from the environment variable
-          URL="${{ vars.CHAIN_URL }}/agoric/vstorage/data/published.vaultFactory.governance"
+          URL="${{ env.CHAIN_URL }}/agoric/vstorage/data/published.vaultFactory.governance"
```

Comment:
What's the difference between env and vars?

Reply:
There are three things here:

Repository environment variables
Environment variables within the workflow
Inputs within the workflow.

`vars.*` refers to repository environment variables. These are values that should be consistent across any workflow run in your repo

`github.event.inputs.*` refers to inputs passed into the workflow

`env.*` refers to environment variables created and available within your workflow